packages/map-template/src/components/LegendDialog/LegendDialog.scss

```diff
@@ -0,0 +1,83 @@
+.background {
```

Comment:
Should we prefix it with something like legend to avoid it being "global"? Can we use BEM?

Reply:
✅